cl/lib/elasticsearch_utils.py

```diff
@@ -108,11 +105,13 @@ def build_term_query(
     :param value: term or terms to find
     :param make_phrase: Whether we should make a match_phrase query for
     TextField filtering.
+    :param slop: Maximum distance between terms in a phrase for a match.
+    Only applicable on make_phrase queries.
     :return: Empty list or list with DSL Match query
     """
 
     if value and make_phrase:
```

Comment:
I think we can refactor this helper and avoid repeating the `value` argument in each `if statement`. We can use an early return and the helper could look like this:

```python
if not value:
        return []

if make_phrase:
    return [Q("match_phrase", **{field: {"query": value, "slop": slop}})]

if isinstance(value, list):
    value = list(filter(None, value))
    return [Q("terms", **{field: value})]

return [Q("term", **{field: value})]
```

Reply:
Good to see your first comment on search stuff.

Reply:
sure!